Sources/App/Modules/User/Models/UserModel.swift

```diff
@@ -25,7 +25,11 @@ final class UserModel: ViperModel {
     /// unique identifier of the model
     @ID() var id: UUID?
     /// email address of the user
-    @Field(key: FieldKeys.email) var email: String
+    @Field(key: FieldKeys.email) var email: String {
+        didSet {
```

Comment:
@mcritz I just ran into following error, when I started feather with no database:
`Fatal error: Cannot access field before it is initialized or fetched: email`
I guess it is because of the use of `didSet` here.

Reply:
I just fixed this issue, via a content middleware, unfortunately there is no easy way to perform a case insensitive filter...

https://github.com/BinaryBirds/feather/commit/ff4a535bf0f45b1ab2480f37ef39f5ff3eef8fc0